app/controllers/organizations/staff/adoption_application_reviews_controller.rb

```diff
@@ -18,6 +20,8 @@ def index
       status_filter = params[:q]["adopter_applications_status_eq"]
       @pets_with_applications = filter_by_application_status(@pets_with_applications, status_filter)
     end
+
+    @pagy, @pets_with_applications = pagy(@pets_with_applications, items: AdopterApplication::REVIEWS_PER_PAGE)
```

Comment:
Can we just set the reviews per page in the controller instead? I see you have put it on the model as a constant, but this is a view related concept and does not really have anything to do with the core domain logic of an AdopterApplication, if you get me? I think specifying it on the controller is preferable.

Reply:
is it okay to set the constants in config/initializers/pagy.rb inside a Pagination module ?  Because I feel that controllers should not be the place to have these constants due to SOC as like models. Would love to hear your thoughts on this

Reply:
I don’t think there’s a need for a constant. Just pass the integer in the pagu method in the controller. That’s how I have seen it done in other apps